RELEASENOTES.md

```diff
@@ -13,6 +13,31 @@ Have new ideas? Need help with setting up your node? Come visit us [here](https:
 
 # Releases
 
+## Version 1.8.3
+This is the 1.8.3 release of `rippled`, the reference implementation of the XRP Ledger protocol.
+
+This release implements changes that improve the syncing performance of peers on the network, adds countermeasures to several routines involving LZ4 to defend against CVE-2021-3520, corrects a minor technical flaw that would result in the server not using a cache for nodestore operations, and adjusts tunable values to optimize disk I/O.
+
+### Summary of Issues
+Recently, servers in the XRP Ledger network have been taking a really long time to sync back to the network after the restart. This is one of several releases which will be made to improve on this issue. 
+
+
+### Bug Fixes
+
+- **Parallel ledger loader & I/O performance improvements**: This commit makes several changes that, together, should decrease the time needed for a server to sync to the network. To make full use of this change, `rippled` needs to be running a high IOPS machine and operators need to explicitly enable this by adding the following to their config file, under the `[node_db]` stanza:
+
+    [node_db]
+    ...
```

Comment:
```suggestion
    # ...
```
Just makes it clear that the ... is an ellipsis and should not be added to the config file literally